js/main.js

```diff
@@ -2,11 +2,70 @@ const checkStringLength = (string, maxLength) => string.length < maxLength;
 
 checkStringLength('Короткий комментарий', 20);
 
-const getRandomArbitrary = (min, max) => {
-  if (min < 0 || max < 0 || min >= max) {
+
+const PHOTOVISUAL = [
```

Comment:
Если слов 2 и больше то их необходимо разделять знаком _
```suggestion
const PHOTO_VISUAL = [
```